istio-proxy/istio-proxy.spec

```diff
@@ -3,7 +3,7 @@
 # Build with debug info rpm
 %global with_debug 0
 # Run unit tests
-%global with_tests 1
+%global with_tests 0
```

Comment:
was this intended?

Reply:
Doing this following product. We have additional work that we need to do to add dependencies if we want to run the tests.